README.md

```diff
@@ -11,15 +11,15 @@
 
 Conda-forge release info : [![Conda Downloads](https://img.shields.io/conda/dn/conda-forge/openfisca-core.svg)](https://anaconda.org/conda-forge/openfisca-core)  [![Conda Version](https://img.shields.io/conda/vn/conda-forge/openfisca-core.svg)](https://anaconda.org/conda-forge/openfisca-core)
 
-[OpenFisca](https://openfisca.org/doc/) is a versatile microsimulation free software. Check the [online documentation](https://openfisca.org/doc/) for more details.
+[OpenFisca](https://openfisca.org/doc/) is a versatile microsimulation-free software. Check the [online documentation](https://openfisca.org/doc/) for more details.
```

Comment:
I'm not sure I fully understand this. With the hyphen, I read it as “a versatile software that is free of microsimulation” instead of “a free software that does microsimulation”. I'd be happy to get the opinion of native English speakers 🙂